docs/DeveloperGuide.md

```diff
@@ -330,6 +332,18 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 3. User requests to add money owed to a specific person in the list
 4. ContactManager update money owed to the person
 
+**Use case: Add task to a contact**
+
+1. When create a contact, user can add a task to that contact
+2. ContactManager shows a list of persons
+3. User is able to see the task that he/she need to do for the contact
+
+**Use case: Edit task to a contact**
+
+1. User requests to list persons
+2. ContactManager shows a list of persons
+3. User is able to edit the contact to update the task to that contact, if no task, user can put `-`
```

Comment:
@yaololo take note of my fix implemented [here](https://github.com/AY2122S2-TIC4002-F18-4/tp2/pull/63)

Reply:
👍 thanks! @TanChiaChun